R/derive_params_growth_height.R

```diff
@@ -93,14 +92,13 @@
 #' advs <- dm_peds %>%
 #'   select(USUBJID, BRTHDTC, SEX) %>%
 #'   right_join(., vs_peds, by = "USUBJID") %>%
-#'   filter(USUBJID != "PEDS-1010") %>%
 #'   mutate(
 #'     VSDT = ymd(VSDTC),
 #'     BRTHDT = ymd(BRTHDTC)
 #'   ) %>%
 #'   derive_vars_duration(
```

Comment:
`trunc_out = FALSE` can be removed as its default anyway for this function - to simplify the example